<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogExit</class>
 <widget class="QDialog" name="DialogExit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>397</width>
    <height>239</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>397</width>
     <height>239</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="res.qrc">:/img/res/exit_dialog.jpg</pixmap>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_cancel">
   <property name="geometry">
    <rect>
     <x>17</x>
     <y>157</y>
     <width>114</width>
     <height>47</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_just_exit">
   <property name="geometry">
    <rect>
     <x>142</x>
     <y>157</y>
     <width>114</width>
     <height>47</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_save_exit">
   <property name="geometry">
    <rect>
     <x>266</x>
     <y>157</y>
     <width>114</width>
     <height>47</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
